draft-ietf-quic-recovery.md

```diff
@@ -994,11 +994,11 @@ congestion without depending on PTO expiration.
 
 ### Establishing Persistent Congestion
 
-A sender establishes persistent congestion after the receipt of an
-acknowledgement if at least two ack-eliciting packets are declared lost, and:
+A sender MUST establish persistent congestion after the receipt of an
+acknowledgement if two ack-eliciting packets are declared lost, and:
 
 * all packets, across all packet number spaces, sent between the send times of
-  two ack-eliciting packets are declared lost;
+  these two packets are declared lost;
```

Comment:
I know we keep disagreeing about this, but I think paralellism is important.  Above, they're called "two ack-eliciting packets" and below we drop 'ack-eliciting', which I find confusing.

If you don't want to restate ack-eliciting, then I'd suggest "if two packets that are ack-eliciting are declared lost,"...